Localization/en-US/home.po

```diff
@@ -0,0 +1,110 @@
+# Schema-Version: 1.0
+# URL: https://foldingathome.org/
+# Source-Language: en-US
+# Source-Version: 1.0
+# Source-Last-Modified: 20200705
+# Target-Language:
+# Target-Version:
+# Target-Last_Modified:
+
+msgid "I AM"
+msgstr ""
+
+msgid "One"
+msgstr ""
+
+msgid "IN A MILLION"
+msgstr ""
+
+msgid "Regardless if you are already folding or haven?t heard a word about it before, we need your help to reach our goal ? which is 1 million folders."
+msgstr ""
+
+msgid "Start folding now"
+msgstr ""
+
+msgid "[Hide this and watch the video.](#)"
+msgstr ""
+
+msgid "START FOLDING NOW"
+msgstr ""
+
+msgid "Find the version of the software you prefer and get started. Downloading Folding@home is completely free, easy to install and safe to use."
+msgstr ""
+
+msgid "Available for:"
+msgstr ""
+
+msgid "Linux"
+msgstr ""
+
+msgid "Windows"
+msgstr ""
+
+msgid "Mac"
+msgstr ""
+
+msgid "[Start folding](https://foldingathome.org/start-folding/)"
+msgstr ""
+
+msgid "Latest Posts"
+msgstr ""
+
+msgid "[The SARS-CoV-2 nucleocapsid protein is dynamic, disordered, and phase separates with RNA.](https://foldingathome.org/2020/06/26/the-sars-cov-2-nucleocapsid-protein-is-dynamic-disordered-and-phase-separates-with-rna/)"
+msgstr ""
+
+msgid "[Conformational distributions of isolated myosin motor domains encode their mechanochemical properties.](https://foldingathome.org/2020/06/04/conformational-distributions-of-isolated-myosin-motor-domains-encode-their-mechanochemical-properties/)"
+msgstr ""
+
+msgid "[The Covid Moonshot](https://foldingathome.org/2020/05/28/the-covid-moonshot/)"
+msgstr ""
+
+msgid "JOIN THE COMMUNITY"
+msgstr ""
+
+msgid "The Folding@Home community is spread over countless forums, websites and social media."
+msgstr ""
+
+msgid "Here?s a list of our official channels."
+msgstr ""
+
+msgid "For tech support:"
+msgstr ""
+
+msgid "foldingforum.org"
+msgstr ""
+
+msgid "[Facebook](https://www.facebook.com/Foldinghome-136059519794607/)"
+msgstr ""
+
+msgid "[Twitter](https://twitter.com/foldingathome)"
+msgstr ""
+
+msgid "1 in a million"
+msgstr ""
+
+msgid "WHY WE NEED YOU"
+msgstr ""
+
+msgid "Folding@home is a project focused on disease research. The problems we?re solving require so many computer calcul�ations ? and we need your help to find the cures!"
+msgstr ""
+
+msgid "Fight diseases Together with us."
+msgstr ""
+
+msgid "The Folding@home software allows you to share your unused computer power with us ? so that we can research even more potential cures."
+msgstr ""
+
+msgid "FOLDING?"
+msgstr ""
+
+msgid "Folding refers to the way human protein folds in the cells that make up your body. We rely on the proteins to keep us healthy and they assemble themselves by folding. But when they misfold, there can be serious consequences to a person?s health."
+msgstr ""
+
+msgid "[Learn more](http://foldingathome.org/diseases/)"
```

Comment:
```suggestion
msgid "Learn more"
```

Again, probably better to only have the string here? Maybe add a comment to explain where this goes?